che-incubator/che-code
# chore: Fix i18n conflicts

### What does this PR do?
The upstream changes conflict with the PR https://github.com/che-incubator/che-code/pull/178 changes.
The current PR fixes i18n conflicts.

### What issues does this PR fix?

https://github.com/eclipse/che/issues/22037

### How to test this PR?
The changes are related to localization tooling for Visual Studio Code. At the moment this functionality doesn't work for Che Code (please see https://github.com/eclipse/che/issues/21963), so there is no way to check it.
